use-cases/microsegmentation.mdx

```diff
@@ -1,17 +1,441 @@
+
+If one or more network policies apply to a pod containing egress rules, then only the egress traffic specifically allowed by those policies is allowed.
+
+Policy Tiers
+Calico Cloud and Calico Enterprise allow hierarchical grouping of policies into tiers.
+Policy Tiers allow enforcement of higher-precedence policies that cannot be circumvented by other teams.
+Policy tiers are evaluated based on order, as are policies within each tier.
+Graphically, policies are evaluated from left to right, top to bottom.
+RBAC for each tier can be defined to restrict who can interact with each tier.
+
+While Calico Open Source does not support policy tiers, you can use RBAC to control how different users can shape cluster security.
+
+Staged Policies
+These rules are used to preview network behavior and do not to enforce network traffic, and can be applied to both network or global policies.
+
+Staged policy lets you test the traffic impact of the policy as if it were enforced, but without changing traffic flow.
+You can also preview the impacts of a staged policy on existing traffic.
+By verifying that correct flows are allowed and denied before enforcement, you can minimize misconfiguration and potential network disruption.
+
+## Implementing microsegmentation with network policies
+
+To implement microsegmentation, you should follow a structured and repeatable approach to increase likelihood of success.
+These can be summarized as four broad steps:
+
+1. [Identify the security domains](./microsegmentation#identification-of-security-domains) for which microsegmentation will be enforced, who will be responsible for them, who or which services need access to those security domains.
+
+2. [Define a policy](./microsegmentation#developing-policy-framework) model using documented microservice communication for your application(s) or by analyzing traffic flows.
+   When defining policies you should also consider: the scope of policies (global/namespace), who will be writing and applying the policies, and policy order (or tiers).
+
+3. [Author and deploy network policies](./microsegmentation#deploy-security-policies).
+   Once all of the correct allow policies are in place, stage a [default deny policy](./microsegmentation#enforce-default-deny).
+   You may want to identify a low-impact application or security domain first to understand and evaluate the process before prioritizing segmentation of critical security domains.
+
+4. [Re-assess any flows](./microsegmentation#monitor-review-refine) or new applications that may require policy remediation before enforcing a default-deny.
+   In Project Calico, where staged policies are not supported, enforce a default deny in a staging environment to correct any policies prior to enforcing in production.
+
+### Identification of security domains
+
+When identifying security domains or applications to secure, you may also want to consider application criticality and compliance requirements.
```

Comment:
I have tried to add more information through [L217-L223](https://github.com/mapgirll/docs/commit/6ebae65a125364605cb7b963c02a1204b336306c#diff-ca03da69780c75a2beea8bd7ac077f9ad4749605a18a5b6f60a46743936dc68fR217-R223).

I imagine if someone's got this far through the doc they're reading it because they have an idea of what needs securing in their organization. When I listened to customer call recordings about microsegmentation the customer already knew roughly what they needed to secure, they just needed help with how. My assumption for the target audience is that this is sufficient.